addons/l10n_ae/models/invoice.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    # Technical field to show rate on invoice report
+    l10n_ae_invoice_rate = fields.Float(compute='_compute_rate', string='Rate on Invoice Date', digits=(12, 6))
+    l10n_ae_amount_tax_signed = fields.Monetary(compute='_compute_rate', string='TaxAmount in AED', currency_field='company_currency_id')
+
+    @api.depends('date_invoice', 'amount_tax', 'currency_id', 'company_id')
```

Comment:
apply same dependent fields which are on _compute_amount of account_invoice model. see: https://github.com/odoo/odoo/blob/master/addons/account/models/account_invoice.py#L52